<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderControl</class>
 <widget class="QFrame" name="RenderControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="translate_btn">
       <property name="text">
        <string>translate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rotate_btn">
       <property name="text">
        <string>rotate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scale_btn">
       <property name="text">
        <string>scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="transform_btn">
       <property name="text">
        <string>transform</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
